package/MDAnalysis/coordinates/base.py

```diff
@@ -1692,6 +1692,70 @@ def __repr__(self):
             natoms=self.n_atoms
         ))
 
+    def timeseries(self, asel=None, start=None, stop=None, step=None,
```

Comment:
This is cool, but maybe it should be called `coordinates_timeseries`? unless it's obvious that the only timeseries of interest is the coordinates?  I also think maybe the function should be made more general to be a timeseries of *all* time data, so box info, velocities etc?  This would make the `Reader -> MemoryReader` transformation trivial.  So returning something like a numpy structured array? Not sure what a good container for this is though...

Reply:
I know, it's not general enough at the moment. However, it is consistent with the MemoryReader https://github.com/MDAnalysis/mdanalysis/blob/f83498ec6b5cc546a0d75598a4d1d6ede6fe2782/package/MDAnalysis/coordinates/memory.py#L475-L478